pkg/volume/iscsi/iscsi_util.go

```diff
@@ -133,16 +188,28 @@ func (util *ISCSIUtil) AttachDisk(b iscsiDiskMounter) error {
 		}
 		exist := waitForPathToExist(devicePath, 1, iscsiTransport)
```

Comment:
Can this `waitForPathToExist` ever succeed if we know that CHAP is enabled and we did not set username / password? IMO it adds unnecessary 1 second delay.